version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+
+## I Nostri Standard
+
+Esempi di comportamento che contribuiscono a creare un ambiente sociale positivo includono:
+
+* Usare un linguaggio accogliente e comprensibile
+* Essere rispettosi di punti di vista ed esperienza differenti
+* Accettare graziosamente criticismi costruttivi
+* Concentrarsi su cosa è meglio per la comunità
+* Mostrare empatia verso altri membri della comunità
+
+Esempi di comportamento inaccettabile da parte dei partecipanti includono:
+
+* L'uso di linguaggio o immagini sessuali e attenzioni o advances sgraditi
+* Comportamento da troll o commenti offensivi/dispregiativi, e attacchi personali o politici
+* Molestie pubbliche o private
+* Pubblicazione di informazioni private di un individuo, quali l'indirizzo reale e/o elettronico, senza l'esplicito consenso
+* Altre condotte che possono ragionevolmente essere considerate inappropriate in un ambiente professionale
+
+## Le Nostre Responsabilità
+
+I mantenitori del progetto sono responsabili nel chiarire gli standard di comportamento accettabile e sono supposti di prendere azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.
+
+I responsabili del progetto hanno il diritto e la responsabilità di rimuovere, modificare, o cancellare commenti, commit, codice, modifiche del wiki, ticket, ed altri contributi che non sono in linea con questo Codice di Comportamento, o di bandire temporaneamente o permanentemente qualsiasi collaboratore per altri comportamenti che verranno ritenuti inappropriati, intimidatori, offensivi o dannosi.
+
+## Ambito
+
+Questo Codice di Comportamento è applicabile sia al progetto online che agli spazi pubblici quando un individuo rappresenta il progetto stesso o la sua comunità. Esempi di rappresentazione di un progetto o una comunità includono l'uso di un indirizzo e-mail ufficiale del progetto, il post in un account ufficiale di un social media, o l'agire come un rappresentante designato in un evento offline o online. La rappresentazione di un progetto può essere ulteriormente definito e chiarificato dai manutentori del progetto.
+
+## Applicazione
+
+Casi di comportamento ingiurioso, molesto o altrimenti inaccettabile possono essere riportati contattando il team del progetto tramite [INSERIRE INDIRIZZO EMAIL]. Tutti i reclami saranno revisionati ed indagati e risulteranno in una risposta ritenuta necessaria ed appropriata alle circostanze. I team del progetto è obbligato a mantenere riserbo rispetto a chi riporta un caso.
+Ulteriori dettagli di applicazioni specifiche possono essere postati separatamente.
+
+I manutentori del progetto che non seguono o applicano il Codice di Condotta in buona fede possono affrontare ripercussioni temporanee o permanenti in base alla deliberazione di altri membri della direzione del progetto.
```

Comment:
I **responsabili** del progetto che non seguono o applicano il **Patto di Condotta** in buona fede possono affrontare ripercussioni temporanee o permanenti in base alla deliberazione **degli** altri membri della direzione del progetto.

- manutentori -> responsabili
- Codice di Condotta -> Patto di Condotta
- aggiunto *degli*